Control coverage/Feature/AppService.md

```diff
@@ -2,5 +2,8 @@
+| <b>ControlId:</b><br>Azure_AppService_Config_Disable_Web_Sockets<br><b>DisplayName:</b><br>Web Sockets should be disabled for App Service. <br><b>Description: </b><br> Web Sockets should be disabled for App Service. | <b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web?<br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/webSocketsEnabled  <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> name <br><br> <b> ARM API to get configuration of an App Service slot: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slotName}/config/web? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/webSocketsEnabled| <b>Passed: </b><br>Web sockets is disabled for all slots in App Service.<br><b>Failed: </b><br>Web sockets is enabled for any of the slot in App Service. |
+| <b>ControlId:</b><br>Azure_AppService_Audit_Enable_Logging_and_Monitoring<br><b>DisplayName:</b><br>Monitoring must be enabled for App Service. <br><b>Description: </b><br> Auditing and Monitoring must be enabled for App Service. | <b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web?<br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/detailedErrorLoggingEnabled, properties/httpLoggingEnabled, properties/requestTracingEnabled  <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> name <br><br> <b> ARM API to get configuration of an App Service slot: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slotName}/config/web? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/detailedErrorLoggingEnabled, properties/httpLoggingEnabled, properties/requestTracingEnabled| <b>Passed: </b><br>HTTP Logging, DetailedError Logging and Request Tracing, all three must be eanbled for all slots in App Service.<br><b>Failed: </b><br>HTTP Logging, DetailedError Logging and Request Tracing, any one of the three logging is disabled for any slot of App Service. |
```

Comment:
Typo in "HTTP Logging, DetailedError Logging and Request Tracing, all three must be **eanbled** for all slots in App Service.". **eanbled** to be corrected to **enabled**.

Reply:
done, fixed the typo.